src/tribol/mesh/MfemData.cpp

```diff
@@ -661,7 +661,7 @@ const MfemSubmeshData::UpdateData& MfemSubmeshData::GetUpdateData() const
 
 MfemJacobianData::MfemJacobianData( const MfemMeshData& parent_data, const MfemSubmeshData& submesh_data,
                                     ContactMethod contact_method )
-    : parent_data_{ parent_data }, submesh_data_{ submesh_data }, block_offsets_{ 3 }
+    : parent_data_{ parent_data }, submesh_data_{ submesh_data }, block_offsets_( 3 )
```

Comment:
Comment: It is what it is, but as we talked about this is kind of annoying since you see a mix of braces and then the parenthesis. If you didn't know, you'd have to go to the header to differentiate between the variables and see that mfem has the different constructor convention. I don't think this warrants a comment or anything, but it could be very easy to miss if you didn't know.

Reply:
Agreed, but we might be able to clean some of this up in a future PR.